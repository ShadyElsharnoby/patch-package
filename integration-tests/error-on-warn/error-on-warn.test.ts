import { runIntegrationTest } from "../runIntegrationTest"
runIntegrationTest({
  projectName: "error-on-warn",
  shouldProduceSnapshots: true,
})
