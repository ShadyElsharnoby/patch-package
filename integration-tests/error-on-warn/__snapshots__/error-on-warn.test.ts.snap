// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test error-on-warn: 00: at dev time patch-package warns but returns 0 1`] = `
"SNAPSHOT: at dev time patch-package warns but returns 0
patch-package 0.0.0
Applying patches...
left-pad@1.1.2 âœ”

Warning: patch-package detected a patch file version mismatch

  Don't worry! This is probably fine. The patch was still applied
  successfully. Here's the deets:

  Patch file created for

    left-pad@1.1.2

  applied to

    left-pad@1.1.3
  
  At path
  
    node_modules/left-pad

  This warning is just to give you a heads-up. There is a small chance of
  breakage even though the patch was applied successfully. Make sure the package
  still behaves like you expect (you wrote tests, right?) and then run

    patch-package left-pad

  to update the version in the patch file name and make this warning go away.

---
patch-package finished with 1 warning(s).
END SNAPSHOT"
`;
